src/cultionet/scripts/cultionet.py

```diff
@@ -213,7 +217,7 @@ def persist_dataset(args):
     ref_res_lists = [args.ref_res]
 
     inputs = model_preprocessing.TrainInputs(
-        regions=config['regions'],
```

Comment:
I like the flexibility of being able to use a different source for the region ids. However, pinning a file and opening as a global overrides the existing configuration that works with the example data and with other ongoing projects.

My suggestion would be to add CLI parameters that would allow us to pass a region id file that would override the configuration integer range.

I think something like the following:
```python
if hasattr(args, 'region_id_file'):
    if not Path(args.region_id_file).is_file():
        raise IOError('The id file does not exist')
    id_data = pd.read_csv(args.region_id_file)
    # How do you feel about an expected column name of 'id'?
    regions = id_data['id'].unique().tolist()  
    # Otherwise, 
    # regions = id_data[args.id_column].unique().tolist()

    inputs = model_preprocessing.TrainInputs(....
        regions=regions,
    )
```

Reply:
@jgrss , I agree. Hard-coding the geo_id file in can be problematic.
I really your idea of passing the file in through CLI. I will implement following your suggestion 👍